tools/configure-fwe.sh

```diff
@@ -57,10 +57,18 @@ parse_args() {
             ENDPOINT_URL=$2
             shift
             ;;
+        --certificate)
```

Comment:
Info on these new options needs to be added to the help info.

Reply:
added